vagrant/README.md

```diff
@@ -0,0 +1,75 @@
+# About
+
+This is a fairly minimal Vagrantfile and associated bootstrap scripts for
+setting up a build environment for
+[TrustedFirmware-M](https://www.trustedfirmware.org).
+
+Note that the vagrant virtual machine will contain an independent copy of TF-M
+and the mbed-os-tf-m-regression-tests.
+
+# Howto
+
+### TF-M + Mbed OS Regression Tests
+
+The following quickstart instructions will build you the TF-M regression tests
+for MUSCA_B1 on Mbed OS.
+
+```
+$ cd vagrant
+$ vagrant up
+$ vagrant ssh
+$ cd mbed-os-tf-m-regression-tests
+$ mbed deploy
```

Comment:
Might be good to include `mbed deploy` when vagrant gets provisioned, as this installs lot of modules.

Reply:
It's in the README. Would be convenient to have this working, but the tools are too complicated to allow this currently.